### ISSUE TITLE
Question about arc.js and alchemy

### ISSUE THREAD

**Issue by goodstemy** [daostack/alchemy#79]
I using alchemy to view my own DAO after I created it with arc.js. 
Step by step:

1.  I run Ganache standalone
2. Connecting to Ganache using metamask
3. In alchemy project I run `npm run migrate-ganache`
4. Then `npm run start-ganache`
5. After I execute migration command in arc.js project `npm explore @daostack/arc.js -- npm start migrateContracts`
5. Then I created DAO with this code: 
<img width="506" alt="2018-05-22 19 13 46" src="https://user-images.githubusercontent.com/8494692/40378663-8888056c-5df4-11e8-8c4f-15f8d668bc54.png">
Browser logs:
<img width="984" alt="2018-05-22 19 14 20" src="https://user-images.githubusercontent.com/8494692/40378656-839bbfbc-5df4-11e8-8197-9f772ab01cc2.png">
Alchemy:
<img width="1440" alt="2018-05-22 19 14 34" src="https://user-images.githubusercontent.com/8494692/40378657-83d120f8-5df4-11e8-914b-7889f6ad2e7a.png">

DAO created successfully but I can't see my DAO in alchemy, only Genesis  ☹️

p.s.

I submit issue here by request: https://github.com/daostack/arc.js/issues/227#issuecomment-391111668

**Comment by dev-matan-tsuberi** [comment]
@tibetsprague I think that's because ganache randomizes the `networkId`. The only reason we see `Ganache` in dev is because the `networkId` is fixed by Arc.js when running `ganacheDb` (see https://github.com/daostack/alchemy/pull/41).

**Comment by goodstemy** [comment]
<img width="1552" alt="2018-05-23 17 28 40" src="https://user-images.githubusercontent.com/8494692/40434485-c42431e4-5eae-11e8-8a5e-9ee3fae576a0.png">

I found something strange. I run ganache and connecting to it from arc.js and I have on account 0x62730... 91.968 eth, but in alchemy, I see the similar account with another eth balance. Is alchemy connecting to my ganache with this command `npm run start-ganache`?

**Comment by dev-matan-tsuberi** [comment]
@goodstemy No. This error is because you didn't start [`alchemy-server`](https://github.com/daostack/alchemy-server) for fetching proposal titles.

**Comment by goodstemy** [comment]
Is there a difference in the startup queue between running migrations of arc.js and alchemy?
